R/learner_lme4_classif_glmer.R

```diff
@@ -18,7 +18,24 @@
 #' `r format_bib("bates2010lme4")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' if (requireNamespace("lme4", quietly = TRUE)) {
+#' # Define the Learner and set parameter values
+#'
+#'
+#' learner = lrn("classif.glmer", formula = outcome ~ treatment * visit + (1 | ID))
+#' print(learner)
+#'
+#' # Define a Task
+#' task = as_task_classif(faraway::toenail, target = "outcome")
```

Comment:
why not a standard task? is the faraway package available?

Reply:
I've tried some standard tests but they're not available for this learner, so I use the dataset in the example of function glmer instead.